.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,63 @@
+name: Bug report
+description: Submit a bug report
+labels: ["type-bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          # New to Python?
+
+          The issue tracker isn't the right place to get help.
+          Consider instead:
+
+            - reading the [Python tutorial](https://docs.python.org/3/tutorial/)
+            - posting on [Discourse](https://discuss.python.org/c/users/7)
+            - emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: Bugs in third-party projects (e.g. `requests`) do not belong in the CPython issue tracker
```

```suggestion
      description: Bugs in third-party projects (e.g. `requests`) should be reported to the projects' issue tracker
```
This uses a more affirmative tone and tells the user what to do instead.  If you want to be extra clear, you could add a "..., not here" or "..., not on the CPython issue tracker" at the end.

---

```suggestion
      description: A bug in a third-party project (e.g. `pip` or `requests`) should be reported to that project's issue tracker, not CPython
```